.flake8

```diff
@@ -39,6 +39,7 @@ extend-exclude =
 extend-ignore =
   E203,  # skipped because it annoys black
   WPS300,  # "Found local folder import" -- nothing bad about this
+  WPS305,  # "Found `f` string" -- f strings are preferred for readability
```

Comment:
I would mention that pylint already covers for f-strings.

Reply:
I'm gonna keep the comment lean here.... I suspect the topic of overlap will come up again if and when someone brings WPS forward as an enforced standard for the repo